sdk/dynatrace/azure-resourcemanager-dynatrace/src/test/java/com/azure/resourcemanager/dynatrace/generated/AccountInfoSecureInnerTests.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.resourcemanager.dynatrace.generated;
+
+import com.azure.core.util.BinaryData;
+import com.azure.resourcemanager.dynatrace.fluent.models.AccountInfoSecureInner;
+import org.junit.jupiter.api.Test;
+
+public final class AccountInfoSecureInnerTests {
+    @Test
+    public void testDeserialize() {
+        AccountInfoSecureInner model =
+            BinaryData
+                .fromString("{\"accountId\":\"mhquvgjxp\",\"apiKey\":\"czmehmtzopbsp\",\"regionId\":\"upi\"}")
```

Comment:
CI reported credential check error for `apikey`:
```
A potential secret was detected in 'AccountInfoSecureInnerTests.java':(CSCAN-GENERAL0130 Client Secret / Api Key) Validate file contains secrets, remove, roll credential, and use approved store
```

Should we replace them with appropriate values?

Reply:
For now, just disable generate-tests?

We might later add logic to skip e.g. `apiKey` property (or the model) when generating the mock JSON in codegen. However we don't know exactly what will be flagged and what will not...

Reply:
Or you can just delete the tests folder in the PR.